mashup-example/Qlik-Sense-Desktop/Using-Build-Tool/index.js

```diff
@@ -0,0 +1,67 @@
+import { embed } from '@nebula.js/stardust';
+import table from '@nebula.js/sn-table/core/esm/index';
+
+import connect from './connect';
+// import embed from './configure';
+import './style.css';
+
+(async () => {
+  const app = await connect();
+  // const nuked = embed(app);
+  const nuked = embed(app, {
+    context: {
```

Comment:
Why is this here and in the configuration?

Reply:
Cleaned up